2-Authorization-I/2-call-arm/SPA/src/components/NavigationBar.jsx

```diff
@@ -0,0 +1,85 @@
+import { Nav, Navbar, Dropdown, DropdownButton } from 'react-bootstrap';
+import { AuthenticatedTemplate, UnauthenticatedTemplate, useMsal } from '@azure/msal-react';
+import { loginRequest } from '../authConfig';
+
+export const NavigationBar = () => {
+    const { instance } = useMsal();
+
+    const handleLoginPopup = () => {
+        instance
+            .loginPopup({
+                ...loginRequest,
+                redirectUri: '/redirect.html',
+            })
+            .catch((error) => console.log(error));
+    };
+
+    const handleLoginRedirect = () => {
+        /**
+         * When using popup and silent APIs, we recommend setting the redirectUri to a blank page or a page
```

Comment:
The comments here should move under the `handleLoginPopup`, not `handleLoginRedirect`